application/backend/src/main/resources/apps/typerefinery/components/widgets/search/dialog/.content.json

```diff
@@ -0,0 +1,45 @@
+{
+    "sling:resourceType": "wcm/dialogs/dialog",
+    "tabs": {
+        "sling:resourceType": "wcm/dialogs/components/tabs",
+        "general": {
+            "sling:resourceType": "wcm/dialogs/components/tab",
+            "label": "General",
+            "placeholder": {
+                "sling:resourceType": "wcm/dialogs/components/textfield",
+                "name": "placeholder",
+                "label": "Placeholder",
+                "description": "Please provide placeholder text"
+            },
+            "componentId": {
+                "sling:resourceType": "wcm/dialogs/components/textfield",
+                "name": "componentId",
+                "label": "Component Id",
+                "description": "Please provide component id to be searched"
+            },
+            "componentType": {
+                "sling:resourceType": "wcm/dialogs/components/select",
+                "name": "componentType",
+                "label": "Component Type",
+                "TABLE": {
```

Comment:
search box should not be linked to any other component, just simply raising events with its content.